packages/functions/src/serializer.ts

```diff
@@ -45,7 +45,7 @@ export class Serializer {
     if (data === true || data === false) {
       return data;
     }
-    if (toString.call(data) === '[object String]') {
+    if (Object.prototype.toString.call(data) === '[object String]') {
```

Comment:
Interesting fact: `toString === Object.prototype.toString` resolves to true in all the other browsers except IE11.